### DISCUSSION TITLE
Create a scheduled azure-cost-cli report and save it on blob storage.

### DISCUSSION THREAD

**Discussion by AndTraHTI** [mivano/azure-cost-cli#121] [Ideas]
Hi @mivano, have you considered making an automatic export and then storing it on blob storage?

**Comment by mivano** [comment]
The CLI tool is an ideal candidate to automate this but as part of a workflow system. So something like GitHub actions that will execute on a schedule and export data where a next step will then process this data (like uploading). 

So, there is no direct support for automatic exports and blob integrations, but it is certainly possible today by combining things like the GitHub Actions and az cli.
